suspendmanager.lua

```diff
@@ -43,6 +46,15 @@ REASON_TEXT = {
     [REASON.DEADEND] = 'dead end',
 }
 
+--- Description of suspension
+--- This only cover the reason where suspendmanager actively
+--- suspend jobs
+REASON_DESCRIPTION = {
+    [REASON.RISK_BLOCKING] = 'Risk blocking another',
+    [REASON.ERASE_DESIGNATION] = 'On a tile designation',
+    [REASON.DEADEND] = 'Blocking a dead-end'
```

Comment:
from conversation on discord, suggest:
- May block another build job
- Waiting for carve/smooth/engrave job on same tile
- Blocks another build job

Reply:
I cut off the "on same tile", with the current overlay I have a single line of real estate